README.md

```diff
@@ -1,55 +1,149 @@
+      ```bash
+      sudo npm link
+      ```
 
-   ```bash
-   covid19 stats
-   ```
+And we are ready to go!
 
-The CLI can also return the COVID-19 statistics of a country, if specified, using one of the following commands.
+## Usage
 
-   ```bash
-   covid19 stats --country <countryName>
-   ```
+This CLI is very easy to use.
 
-   ```bash
-   covid19 stats -c <countryName>
-   ```
+*   **covid19**  
+    This is the name of the package that is used to invoke the CLI.  
+    Flags are available for this command.  
 
-The supported formats for `countryName` are ISO2, ISO3 or a single-word name.
+    <table style="height: 5px; width: 600px;" border="black">
 
-## CONTRIBUTING
+    <tbody>
+
+    <tr style="border-color: black;">
+
+    <td style="width: 150px; text-align: center; border-color: black;"><span style="text-decoration: underline;">flag</span></td>
+
+    <td style="width: 150px; text-align: center; border-color: black;"><span style="text-decoration: underline;">alias</span></td>
+
+    <td style="width: 300px; text-align: center;  border-color: black;"><span style="text-decoration: underline;">description</span></td>
+
+    </tr>
+
+    <tr>
+
+    <td style="width: 150px; text-align: center; border-color: black;">--version</td>
+
+    <td style="width: 150px; text-align: center; border-color: black;">-v</td>
+
+    <td style="width: 300px; text-align: center; border-color: black;">Display the current version of the program</td>
+
+    </tr>
+
+    </tbody>
+
+    </table>
+    <br/>
+
+    Example:
+    ```bash
+    covid19 -v
+    ```
+    OR
+    ```bash
+    covid19 --version
+    ```
+
+*   **covid19 stats**  
+    This command returns the statistics of CoVID 19 all over the world.
+
+    Example:
+    ```bash
+    covid19 stats
+    ```
+
+    Flags are available for this command.  
+
+    <table style="height: 5px; width: 600px;" border="black">
+
+    <tbody>
+
+    <tr style="border-color: black;">
+
+    <td style="width: 150px; text-align: center; border-color: black;"><span style="text-decoration: underline;">flag</span></td>
+
+    <td style="width: 150px; text-align: center; border-color: black;"><span style="text-decoration: underline;">alias</span></td>
+
+    <td style="width: 300px; text-align: center;  border-color: black;"><span style="text-decoration: underline;">description</span></td>
+
+    </tr>
+
+    <tr>
+
+    <td style="width: 150px; text-align: center; border-color: black;">--country<br/> &ltcountryName&gt</td>
+
+    <td style="width: 150px; text-align: center; border-color: black;">-c &ltcountryName&gt</td>
+
+    <td style="width: 300px; text-align: center; border-color: black;">Display statistics for a particular country</td>
+
+    </tr>
+
+    </tbody>
+
+    </table>
+    <br/>
+    The supported formats for countryName are ISO2, ISO3 or a single-word name.
+    <br/>
+    A list of ISO Country Codes can be <a href="https://www.iban.com/country-codes" target="_blank">found here</a>
```

Comment:
It's fine to join this statement with the above statement. You can remove the `br` tag above this, as I mentioned earlier.

Reply:
Don't forget to add a "period" at the end of this statement.